Source/towners.cpp

```diff
@@ -898,6 +898,12 @@ bool DebugTalkToTowner(std::string targetName)
 	std::transform(targetName.begin(), targetName.end(), targetName.begin(), [](unsigned char c) { return std::tolower(c); });
 	auto &myPlayer = Players[MyPlayerId];
 	for (auto &towner : TownerInitList) {
+		// prevent going into hellfire territory without hellfire
+		if (towner.type == TOWN_COWFARM && !gbIsHellfire)

```

Comment:
does this also affect the nut and the girl?

Reply:
yes because they are behind in the table